rfd/0159-password-status.md

```diff
@@ -66,6 +68,48 @@ ContactAdmin --> End([End])
 
 For clarity, error cases (such as password mismatch or inability to confirm user identity) were omitted from the above diagram.
 
+We are going to implement the following changes to the WebAuthn flow and APIs:
+
+1. Move the responsibility for verifying the old password to the `ChangePassword` RPC call. This will allow us to decouple the verification from setting the new password in the UI (the old password will not be required when generating the challenge). This verification will be specific to the `ChangePassword` call and will not be a part of the generic `LoginFlow`. The
```

Comment:
Leftover word?

```suggestion
1. Move the responsibility for verifying the old password to the `ChangePassword` RPC call. This will allow us to decouple the verification from setting the new password in the UI (the old password will not be required when generating the challenge). This verification will be specific to the `ChangePassword` call and will not be a part of the generic `LoginFlow`.
```